jquery.time-to.js

```diff
@@ -393,6 +398,10 @@
             digit = data.iSec;
         }
 
+        tc = data.tickCount + 1;
+
+        this.data('tickCount', tc);
```

Comment:
If we don't use `tc` anymore join these two lines to
`this.data('tickCount', data.tickCount + 1);`
And remove `tc` in declaration block.